src/main/java/org/gwtproject/tools/cldr/Processor.java

```diff
@@ -258,46 +226,10 @@ protected void printVersion(PrintWriter pw, GwtLocale locale, String prefix) {
     pw.println(prefix + "DO NOT EDIT - GENERATED FROM CLDR DATA");
   }
 
-  /**
-   * Writes a file containing CLDR version information.
-   * @param path the filename, relative to {@link #I18N_PACKAGE_PATH}.
-   */
-  protected void writeVersionFile(String path, Set<GwtLocale> locales) throws IOException {
-    Map<String, GwtLocale> byName = new HashMap<String, GwtLocale>();
-    for (GwtLocale locale : locales) {
-      String name = locale.getAsString();
-      if (byName.containsKey(name)) {
-        throw new RuntimeException("more than one locale with name: " + name);
-      }
-      byName.put(name, locale);
-    }
-
-    List<String> names = new ArrayList<String>(byName.keySet());
-    Collections.sort(names);
-
-    PrintWriter out = createOutputFile(path);
-    out.println("cldrVersion=" + CLDRFile.GEN_VERSION);
-    out.println();
-    for (String name : names) {
-      Map<String, String> props = localeData.getEntries("version", byName.get(name));
-      List<String> keys = new ArrayList<String>(props.keySet());
-      Collections.sort(keys);
-      for (String key : keys) {
-        if (key.equals("date") || key.equals("type")) {
-          // The date comes from Subversion and depends on the local machine's timezone.
-          // Skip it to make the build deterministic.
-
-          // There is more than one "type" field and it's not obviously useful.
-          continue;
-        }
-        String value = props.get(key);
-        if (!name.isEmpty()) {
-          key = name + "." + key;
-        }
-        out.println(key + "=" + value);
-      }
-    }
-    out.close();
+  protected static AnnotationSpec generatedAnnotation(Class<? extends Processor> processor) {
+    return AnnotationSpec.builder(Generated.class)
+            .addMember("value", "\""+"gwt-cldr-importer : "+processor.getCanonicalName()+", CLDR version : "+CldrVersion.value+"\"")
```

Comment:
this looks a little weird, why break up into two strings? `"\"gwt-cldr-importer : "`